docs/storage/multipathing.md

```diff
@@ -166,7 +174,7 @@ linkStyle 13 stroke:#8C8C8C,stroke-width:2px;
 
 3. If necessary, migrate the VMs active on the XCP-ng host in question to another one.
 4. Reboot the XCP-ng host.
-5. Do the same for all XCP-ng hosts in the pool.
+5. Do the same for all XCP-ng hosts in the pool (step 1. to 4.).
```

Comment:
Step 1 is not necessary, as the file is identical on every host. Also applies to the other section.